.github/workflows/deploy.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: npx eslint --max-warnings=0 services
   unit-test:
     name: Unit Test
-    runs-on: macos-latest
```

Comment:
😮

Reply:
That got changed back!  It was part of the experimenting :). apparently the macos is more stable and 10 times more expensive (which I didn't find out until after I tried it and yes, it worked :) )